### DISCUSSION TITLE
iPad sharing broken

### DISCUSSION THREAD

**Discussion by SlashDevSlash0** [localsend/localsend#2001] [Discussion (English, International)]
Today I received the update to version 1.16.0 on my iPads.

Before getting the update I could pick files from the share-button in third-party-apps or from the files or photos app.

Now there is a short flicker of the display and nothing else happens when trying to do so.

The only way to share files or photos  at the moment is using the app directly.

This happens on two iPads running on iOS 18.1. This is very disappointing for me, cause exactly the same happend after updating LANDrop. which I used before and I was so happy to find localsend...

**Comment by Tienisto** [comment]
Sorry to hear that. I've created #2002 to track this issue.
